### DISCUSSION TITLE
Outdated registry url in docs

### DISCUSSION THREAD

**Discussion by Jolg42** [npm/feedback#102] [Website] [answered]
Hi!

I found an outdated registry url in docs
https://docs.npmjs.com/cli/v6/using-npm/registry
https://docs.npmjs.com/cli/v7/using-npm/registry

>The npm public registry is powered by a CouchDB database, of which there is a public mirror at https://skimdb.npmjs.com/registry. The code for the couchapp is available at https://github.com/npm/npm-registry-couchapp.

I didn't expect to see `https://skimdb.npmjs.com/registry` and expected to see `https://replicate.npmjs.com`

Because `skimdb` id deprecated and only has unscoped packages.

**Comment by Jolg42** [comment, marked as answer]
I looked for the source code of the website but it I guess it is private.

**Reply by MylesBorins** [reply]
I've opened a PR on our internal repo and pinged the registry team to confirm if we should make this change. I'll submit a new response and mark as answer once we resolve the open PR.

As an aside we have a work item on the public roadmap to open source the new docs repo next quarter https://github.com/npm/roadmap/issues/17

**Reply by ethomson** [reply]
FYI - the documentation for `/cli/` pages is in the cli itself, https://github.com/npm/cli already.  I'm going to make this change there, should be taken care of this week.  Thanks for the info @jolg42!

**Reply by Jolg42** [reply]
Awesome! <3
